src/cc1101.cpp

```diff
@@ -626,10 +630,7 @@ void cc1101::getRxFifo(uint16_t Boffs) {           // xFSK
 					MSG_PRINT(F(";R="));
 					MSG_PRINT(RSSI);
 					MSG_PRINT(F(";A="));
-					MSG_PRINT(u_int8_t(freqErr));
-					MSG_PRINT(';');
-					MSG_PRINT(F(";A="));
-					MSG_PRINTtoHEX(freqErr);
+					MSG_PRINT(freqErr);
```

Comment:
Das mit dem Vorzeichen gefällt mir hier nicht, wieso übergeben wir den Wert nicht als unsigned Int?

Reply:
Erstens sehe ich dadurch schon an der RAWMSG, ob die Frequenz zu hoch oder zu niedrig ist:

`RAWMSG     MN;D=C1A4BE40041218FEEEFE0008215632FFF0490000000000000000;R=190;A=-3;`

und zweitens erspart uns das die Umrechnung in der SIGNALduino.pm.